src/data.js

```diff
@@ -9271,6 +9271,30 @@ module.exports = [
       'Golang',
     ],
   },
+  {
+    name: 'Ale Mohamad',
+    description:
+      'iOS Developer. I love to learn and to teach. デベロッパー。',
+    url: 'https://alemohamad.com/uses/',
+    twitter: '@alemohamad',
+    emoji: '✨',
+    country: '🇪🇸',
+    computer: 'apple',
+    phone: 'apple',
```

Comment:
from the CI validation job `"phone" must be one of [iphone, android, windowsphone, flipphone]`